Index: app/build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/build.gradle.kts b/app/build.gradle.kts
--- a/app/build.gradle.kts	(date 1756867137129)
+++ b/app/build.gradle.kts
@@ -12,7 +12,7 @@
         minSdk = 29
         targetSdk = 36
         versionCode = 1
-        versionName = "3.0"
+        versionName = "2.0"
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     }
